automatic/opera/tools/chocolateyInstall.ps1

```diff
@@ -16,7 +17,7 @@ $packageArgs = @{
   checksum64     = '301ea4ece36b68631c90b26c8c3e3f54e4ec23e826febd594398cca508e75819'
   checksumType   = 'sha256'
   checksumType64 = 'sha256'
-  silentArgs     = '/install /silent /launchopera 0 /setdefaultbrowser 0' + $parameters
+  silentArgs     = '/install /silent /launchopera=0 /setdefaultbrowser=0' + $parameters
```

Comment:
Don't this work either way?

I am sure I remember that it worked previously with spaces.

Reply:
I specifically experienced `/allusers 1` failing to provide the expected functionality, and `/allusers=1` successfully providing the expected functionality. Given that the --help examples used an equals sign, and for uniformity, I thought it best to switch from spaces to equals for all arguments with values.

Reply:
fair enough, been a while since I tested so it probably changed (or I just remember wrong)